campaigns/rustreach/how-I-came-to-rust.md

```diff
@@ -0,0 +1,58 @@
+
+## Why I'm staying
+
+I guess you won't be able to draw that much from that story. All the conditions might not apply to you. So I guess it's important to tell you why I am still around. And that's because I think the Rust project gets a lot of things _right_. Some of these things weren't always right and there's always things that need to be fixed. A previous draft of this text had included some of the things that were broken. I decided to remove them, because they are _fixed_. For me, Rust - the project - is based on a couple of things:
+
+* Work-focus. Steves invitation mail to me contained the sentence "we need to productively move forward". That hit the right note. Working on the Rust project is not a merit badge for me, it's a place where I want to spend my effort for a while.
+* Structure, and structural change, is seen as a solution. The organisation structure of the project frequently changes, on need.
+* A practical approach to organisation. Because we have a weak hierarchy, special-interest groups can form very quickly and be shut down as quickly. Still, being able to assign names (e.g. "This is the code generation working group") makes people adressable.
+* Trust. The teams have autonomy in what they want to do. The core doesn't give instructions, the core team holds ties together. After being within the project for a while and noticing that people _do_ read and note your email, even if they don't always reply has been very good for me.
+* No single-person "owner" of Rust (BDFL): I like that the Rust project is an ongoing conversation. From language features, over community action to where we want to apply our power. And it somehow works.
+* The mutual understanding that issues will always exist. We're a global project of mostly amateurs. We might do similar things in our day-job, but most of us are _not_ working on a global async project for free as their main profession. The list of problems we had in the last three years is interesting and many, I have never faced before.
+* Rust has working complaint structures. If I have a problem with how things are going, I know where to send them. I know someone will reply with appropriate urgency. (A week or so, we are hobbyists)
+* Issues, especially social ones, will be dealt with. At some point, obviously, but we work on things. I like the way the project is improving at a steady pace and not rushing things.
+* Actively working for a good tone and balance. Rust has people of many cultures and communication issues arise from that. I don't want to make that sound like I'm the master of this, I have been called out multiple times for ways of arguing or tone. This is great, it starts a conversation and allows to check behaviour.
+* No "you must be this tall to be a project member". If you want to start working on something of project capacity, let us know!
```

Comment:
I'm not sure what "something of project capacity" means, perhaps it means "something on the project, in any capacity"?